Header header
time timeout
geometry_msgs/Polygon[] surface_polygons
string[] models
---
SceneHypothesis scene
